File: Lang52a/lang52a.jl


print("Enter Length: ")
length = readline()
print("Enter Width: ")
width = readline()
n1 = parse(Int64, width)
n2 = parse(Int64, length)
perimeter = n2 + n1 + n2 + n1
area = n1 * n2
println("This is the area: $area")
println("this is the perimeter: $perimeter")

File: Lang52a-in-5-languages/lang52a.go
package main

import (
  "fmt"
)
func main() {
	fmt.Print("Enter length:")
	var length int
	fmt.Scanln(&length)
	fmt.Print("Enter width:")
	var width int
	fmt.Scanln(&width)
	var area = length * width
	var perimeter = (length + width) * 2
	var A = fmt.Sprint(area)
	var P = fmt.Sprint(perimeter)
	fmt.Println("This is the area:", A)
	fmt.Println("This is the perimeter:", P)
}



